intel-sgx/fortanix-sgx-tools/src/bin/ftxsgx-elf2sgxs.rs

```diff
@@ -797,6 +810,16 @@ fn main() {
                 .required(true)
                 .help("Specify stack size"),
         )
+        .arg(
+            Arg::with_name("unmapped-memory-size")
+                .short("U")
+                .long("unmapped-memory-size")
+                .value_name("BYTES")
+                .validator(u64::validate_arg)
+                .required(false)
+                .default_value("0")
+                .help("Specify the (minimum) size of unmapped memory (useable on SGXv2 capable processors only)"),
```

Comment:
I wonder, should we clarify why a user would be interested in this option? E.g.

> Specifying an unmapped memory size allows dynamically allocating additional heap space on SGXv2 capable processors

I understand on the other hand that maybe we don't want to advertise it if the whole feature set is not ready yet; not sure.
What do you think?

Looks good to me either way :+1:

Reply:
Maybe we should even call it dynamic memory or something.

Reply:
Good point about not adding it to master yet. Will mark the PR as draft again